block/manager_test.go

```diff
@@ -245,84 +232,15 @@ func TestIsDAIncluded(t *testing.T) {
 	require.True(m.IsDAIncluded(hash))
 }
 
-func TestSubmitBlocksToMockDA(t *testing.T) {
```

Comment:
will readd this in a follow up pr (https://github.com/rollkit/rollkit/pull/2209)